src/main.cpp

```diff
@@ -142,6 +143,26 @@ void adjustScaleFactor(CmdlineArgs* pArgs) {
     }
 }
 
+void getStyleOverride(CmdlineArgs* pArgs) {
```

Comment:
This dos not rellay "get" anything. Do we have a better name?

Reply:
Renamed to `applyStyleOverride`